docs/index.md

```diff
@@ -1,31 +1,23 @@
 Zanata Server
 =============
+Please visit [our about page](http://zanata.org/about/)
 
-TBD
-
-Features
---------
-
-- TBD
 
 Installation
 ------------
 
-<!-- For installation instructions please see the [Installation Section](installation) -->
+[Installation Section](installation)
 
 Contribute
 ----------
 
-- Issue Tracker: http://bugzilla.redhat.com
-- Source Code: github.com/zanata/zanata-server
+- Issue Tracker: [Bugzilla](http://bugzilla.redhat.com/buglist.cgi?product=Zanata)
+- Source Code: [GitHub](http://github.com/zanata)
 
 Support
 -------
-
-If you are having issues, please let us know.
-We have a mailing list located at: zanata-users@redhat.com
+Click [here](http://zanata.org/help/) for help and support
```

Comment:
We should avoid having link text like "here" since it causes big problems for accessibility. We can do something like:

```
For help and support options, see [Help | Zanata](...).
```